controller/embedding/manager.py

```diff
@@ -134,4 +104,110 @@ def get_embedding_name(
         raise ValueError("attribute not found")
     attribute_name = attribute_item.name
 
-    return f"{attribute_name}-{embedding_type}-{embedding_handle}"
+    name = f"{attribute_name}-{embedding_type}-{platform}"
+
+    if model:
+        name += f"-{model}"
+
+    if apiToken:
+        name += f"-{apiToken[:8]}...{apiToken[-4:]}"
```

Comment:
Because we do not only use openAI but cohere too I thought we may should have some more tokens in it but yeah can make a 3 out of the 8 if you would prefer it